src/Stripe.net/Entities/StripeLegalEntityVerification.cs

```diff
@@ -13,10 +13,14 @@ public class StripeLegalEntityVerification : StripeEntity
 
         #region Expandable Document
         public string DocumentId { get; set; }
+        public string DocumentIdBack { get; set; }
```

Comment:
I had thought we might want to group it into the same region but tbh its been a while since I've worked with c#

Reply:
Seems like the purpose is [purely an IDE feature](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-region). Would you as a developer want to collapse both of these together always or would there be a reason to have this in its own region? Since they're so related I would think we'd want them together but idk.

Reply:
Okay I just placed it in its own region to follow convention. ✅